### PR TITLE
Console bridge logging

### PR DESCRIPTION
**Pull request by reinzor**
Addresses https://github.com/ros-drivers/odva_ethernetip/issues/13

### CODE REVIEW

**Comment by reinzor** [comment]
Travis seems to fail on 14.04 due to `CONSOLE_BRIDGE_` prefix

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 sudo: required
-dist: trusty
+dist: xenial
```
Comment (gavanderhoorn) [inline .travis.yml]: This is not a valid value for the `dist` key. `trusty` is the most recent one.

Comment (reinzor) [reply .travis.yml]: Hmm, why is this building then:

https://travis-ci.org/ros-drivers/odva_ethernetip/jobs/449282634

Comment (reinzor) [reply .travis.yml]: Hmm, I see it is only experimental and should not be used ...

**Comment by reinzor** [comment]
Not compatible with indigo, I will open a PR in kinetic-devel

**Pull request closed without merging**